### ISSUE TITLE
Cover art files should use larger files first

### ISSUE THREAD

**Issue by eharris** [airsonic/airsonic#711]
In my music library the cover art is stored in different filenames based on the name of the album.  So a standard name like cover.jpg in the "Cover art files" portion of the config usually doesn't get them, and instead the fallback to just jpg is needed.

Unfortunately I usually also have several sizes of the cover art, large, medium, small, and thumb.  It seems as though airsonic just uses whatever jpg it finds first, which sometimes happens to be the undesirable thumbnail or other smaller sizes, when it would be a much better heuristic to use the largest jpg.

One potential workaround that might help some of the time would be if there was a way to wildcard the filenames in the config, so I could specify something like "*_large.jpg" earlier in the list than the catch-all jpg match.  Is there a way to do that?

**Comment by jooola** [comment]
You could maybe try to exclude using the Exclude pattern function : 
![image](https://user-images.githubusercontent.com/19195485/36978270-34c4217a-2084-11e8-9c87-cd3fac18fd23.png)
And I don't know if the cover art files options can accept wildcard `*` but did you tried to only add `_large.jpg`
![image](https://user-images.githubusercontent.com/19195485/36978328-701ba162-2084-11e8-9ad7-5ed300524d2c.png)

**Comment by eharris** [comment]
I don't really want to exclude anything, because if there's nothing else better, I would still want to use the smaller size images.

I haven't tried just adding the ending suffix, I will try that out.

**Comment by jooola** [comment]
Yes also maybe they are a precedence in the Cover Art files list. Unsure about that.

**Comment by eharris** [comment]
It either looks like there is no precedence in the ordering of the files it selects for the cover art, or there is no pattern/stub matching.  I just wiped out my database to get it to re-detect cover art, and it is often picking any old .jpg in the directory rather than the files I have specified (assuming/hoping that there was precedence in the album art detection).

My config specifies the following for Cover Art Files:
```Front_Large Front_Small Back_Large Back_Small jpg gif png```

I want it to use the ```Front_Large.(gif|jpg|png)``` first, followed by the others in that order.

Unfortunately it is very difficult to try to test different configurations and what airsonic uses for the album art since there seems to be no reasonably reliable way to force airsonic to update the cover art other than wiping the DB (see issue #649).

**Comment by Faeb35** [comment]
I'm also struggeling with this...
I have a lot of albums, with multiple images in the folders. As example I have the following images:
dr. dre & snoop dogg - the wash (soundtrack) - back.jpg
dr. dre & snoop dogg - the wash (soundtrack) - cd.jpg
dr. dre & snoop dogg - the wash (soundtrack) - front.jpg
dr. dre & snoop dogg - the wash (soundtrack) - inlay.jpg
dr. dre & snoop dogg - the wash (soundtrack) - inside.jpg

Airsonic always seams to take the first image (which matchs the configed settings), according to the alphabet. So , the ...back.jpg is used as cover.
Bummer: In my ID3 tags all my covers are configured correctly.

Would be nice to have the following possibilities in Airsonic:

1. Wildcards would be accepted in "Cover art files" setting
2. Have an option to prefere ID3 covers over file covers

Current workaround: 
Rename all images you like to have as main-cover to "cover.jpg" (or png, gif). This does the trick.
